go.mod

```diff
@@ -1,8 +1,14 @@
-module github.com/Pixel8Labs/go-solidity-sha3
+module github.com/pixel8Labs/go-solidity-sha3
```

Comment:
This one not typo ser ? lowercase 'p'

Reply:
hmm, it's auto generated

Reply:
ok anw let's rename it to lowercase 'p' instead, our repos are consistently with lowercase p